packages/widgets/src/components/UserProfile/UserProfile.tsx

```diff
@@ -0,0 +1,54 @@
+import { Link } from 'react-router-dom';
+import { useTheme } from '@emotion/react';
+import {
+  Button,
+  Icon,
+  Popover,
+  PopoverContent,
+  PopoverDescription,
+  PopoverHeading,
+  PopoverTrigger,
+} from '@ssa-ui-kit/core';
+import { UserProfileProps } from './types';
+import {
+  UserInfo,
+  ContentWrapper,
+  LogoutWrapper,
+  ResetBtnStyles,
+  CustomButton,
+} from './styles';
+
+export const UserProfile = ({
+  name,
+  email,
+  trigger,
+  link,
+}: UserProfileProps) => {
+  const theme = useTheme();
+  return (
+    <Popover placement="bottom-end">
+      <PopoverTrigger css={ResetBtnStyles}>{trigger}</PopoverTrigger>
+      <PopoverContent css={ContentWrapper}>
+        <div css={UserInfo}>
+          <PopoverHeading variant="h5" weight="bold">
+            {name}
+          </PopoverHeading>
+          <PopoverDescription
+            css={{ fontSize: 14, marginTop: 6 }}
+            weight="regular"
+            color={theme.colors.greyDropdownFocused}>
+            {email}
+          </PopoverDescription>
+        </div>
+        <div css={LogoutWrapper}>
+          <Link to={link} css={{ textDecoration: 'none' }}>
```

Comment:
I don't think "Log out" should be a link in our case. A button is enough.

Reply:
Done